khal/aux.py

```diff
@@ -139,6 +139,20 @@ def generate_random_uid():
     choice = string.ascii_uppercase + string.digits
     return ''.join([random.choice(choice) for _ in range(36)])
 
+def construct_event_list_from_string(ics_as_string):
+    """Construct a list with icalendar.event types with a random UID"""
+    try:
+        events = icalendar.Calendar.from_ical(ics_as_string)
+    except ValueError:
+        logger.fatal("Please specify a ics file")
```

Comment:
I checked it with an ics file that had <code>END:VEVEN</code> instead of <code>END:VEVENT</code> and icalendar imported it without throwing any errors.